label_studio_ml/examples/bert_classifier/model.py

```diff
@@ -131,7 +131,7 @@ def fit(self, event, data, **additional_params):
 
         logger.info(f"Downloaded {len(tasks)} labeled tasks from Label Studio")
         logger.debug(f"Tasks: {tasks}")
-        if len(tasks) % self.START_TRAINING_EACH_N_UPDATES != 0:
+        if len(tasks) % self.START_TRAINING_EACH_N_UPDATES != 0 and event != 'START_TRAINING':
```

Comment:
Should these conditionals be `or` instead of `and` ?

Reply:
This condition is for the skip of training and equal to this one:

```
if not (len(tasks) % self.START_TRAINING_EACH_N_UPDATES == 0 or event == 'START_TRAINING')
=> 
if len(tasks) % self.START_TRAINING_EACH_N_UPDATES != 0 and event != 'START_TRAINING':
```